folium/map.py

```diff
@@ -378,15 +375,16 @@ def __init__(self, location, popup=None, icon=None):
         Example
         -------
         >>>map.simple_marker(location=[45.5, -122.3], popup='Portland, OR')
-        >>>map.simple_marker(location=[45.5, -122.3], popup=(vis, 'vis.json'))
-
+        >>>map.simple_marker(location=[45.5, -122.3], popup=folium.Popup('Portland, OR'))
```

Comment:
Add a space `>>> map` so we can get this properly rendered later when building the docs.